Tests/ClientRuntimeTests/ClientRuntimeTests/MiddlewareTests/OperationStackTests.swift

```diff
@@ -13,13 +13,6 @@ class OperationStackTests: HttpRequestTestBase {
 
     func testMiddlewareInjectableInit() async throws {
         var currExpectCount = 1
-        let defaultTimeout = 2.0
-        let expectInitializeMiddleware = expectation(description: "initializeMiddleware")
```

Comment:
I faced a similar issue that expectation is not available on Linux. I solved it by using `XCTestExpectation` instead. We can use something similar if we need expectation in future.

Reply:
Expectations are available but the asynchronous-safe wait function (`await fulfillment(of:timeout`) is missing on Linux.  No idea why it's not there.

And in any case, these tests are async but not concurrent (i.e. no new `Task`s are created) and all code is guaranteed to execute before the test ends.  No expectations required to verify that all steps are completed in the expected order.